src/components/PhotoAlbum.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Photos from './Photos';
+import styles from './style.css';
+
+export default function PhotoAlbum({ title, photos }) {
+  return (
+    <container className={styles.PhotoAlbum}>
+      <h2>{title}</h2>
+      <Photos photos={photos} />
+    </container>
+  );
+}
+
+PhotoAlbum.propTypes = {
+  title: PropTypes.string.isRequired,
```

Comment:
How could you give PhotoAlbum a default title in case a title wasn't passed to this component?